### PR TITLE
Update frequency-management.adoc

### PR DESCRIPTION
**Pull request by timg236**

### CODE REVIEW

**Review by aallan — APPROVED** [review]
LGTM. Merging to develop. Will push to production and the main site later today.

**Review by Mandog1120 — COMMENTED** [review]
a40f49aa7a1a02bad7a47732ade091ffc1ddcf0b

**Comment by Mandog1120** [comment]
85d1568fad48f45639d8b5a454adbe7e557075df

File: documentation/asciidoc/computers/raspberry-pi/frequency-management.adoc [hunk 1]
```diff
@@ -48,4 +51,4 @@ $ vcgencmd measure_temp
 
 === Adding a Heatsinks
 
-Whilst heatsinks are not necessary to prevent overheating damage to the SoC — the thermal throttling mechanism handles that — a heatsink or small fan will help if you wish to reduce the amount of thermal throttling that takes place. Depending on the exact circumstances, mounting the Pi vertically can also help with heat dissipation, as doing so can improve air flow.
\ No newline at end of file
+Whilst heatsinks are not necessary to prevent overheating damage to the SoC — the thermal throttling mechanism handles that — a heatsink or small fan will help if you wish to reduce the amount of thermal throttling that takes place. Depending on the exact circumstances, mounting the Pi vertically can also help with heat dissipation, as doing so can improve air flow.
```
Comment (Mandog1120) [inline documentation/asciidoc/computers/raspberry-pi/frequency-management.adoc:54]: documentation/asciidoc/computers/raspberry-pi/frequency-management.adoc

**Comment by aallan** [comment]
> [a40f49a](https://github.com/raspberrypi/documentation/commit/a40f49aa7a1a02bad7a47732ade091ffc1ddcf0b)

This is a two year old closed issue. Please don't comment here, open a new issue if there is a problem.

**Review by Mandog1120 — COMMENTED** [review]
85d1568fad48f45639d8b5a454adbe7e557075df

**Pull request merged**